emoji/slack_emoji_client.py

```diff
@@ -26,4 +29,21 @@ def remove_emoji(self, name):
         return self.client.api_call("emoji.remove", data={"name": name, "mode": "data"})
 
     def emoji_dict(self):
-        return self.client.emoji_list().data['emoji']
+        if self._emoji_dict is None:
+            self._emoji_dict = self.client.emoji_list().data['emoji']
+
+        return self._emoji_dict
+
+    def hashes_for_emoji_images(self):
+        if self._hashes_for_emoji_images is None:
+            self._hashes_for_emoji_images = {}
+            destination_emoji_with_urls = { name: val for name, val in self.emoji_dict().items() if not val.startswith("alias:") }
```

Comment:
do we update this dict after more emoji are transferred?

Reply:
No, support for multiple additions in the same request will be a new PR.  Still need to figure out the best way to do this.